### DISCUSSION TITLE
Does the Postgres docker manifest exist?

### DISCUSSION THREAD

**Discussion by shooftie** [SamTV12345/PodFetch#418] [Q&A] [answered]
Using the Postgres`docker-compose` from the [README](https://github.com/SamTV12345/PodFetch#docker-compose-1) (samuel19982/podfetch:postgres) suggests that the image that does not exist.

> failed to pull images of the stack: postgres Pulling podfetch Pulling postgres Pulled podfetch Error Error response from daemon: manifest for samuel19982/podfetch:postgres not found: manifest unknown: manifest unknown

Looking for it on [Docker Hub](https://hub.docker.com/r/samuel19982/podfetch/tags), it would appear that the last version that mentions Postgres was [v4.4.0-postgres-b2febd8](https://hub.docker.com/layers/samuel19982/podfetch/v4.4.0-postgres-b2febd8/images/sha256-55aaf228859e66bc3032e984a3420268c5217e92aba6318f0ab07c14d50eda22?context=explore).

Using the latest image with the Postgres instructions obviously fails, too.

Am I missing a trick here?

**Comment by SamTV12345** [comment, marked as answer]
Hi. Thanks for reporting this issue. I forgot to adapt the docker-compose to the changes last week. You can use the now updated docker-compose-postgres.yml.

**Reply by shooftie** [reply]
Fantastic work. I jumped ship from PodGrab because development seemed to stall. Not so with this project!

I have a number of feature requests/suggestions. As soon as I get a chance I will throw them your way.
